bin/sandbox

```diff
@@ -45,8 +45,20 @@ sandbox_path="./${sandbox_name}"
 
 if [[ -n $SKIP_SPECS ]]; then
   GENERATOR_SKIP_SPECS_ARGUMENT='--skip-specs'
+  rspec_rails_gem_line=''
 else
   GENERATOR_SKIP_SPECS_ARGUMENT=''
+  rspec_rails_gem_line="gem 'rspec-rails', groups: [:development, :test]"
+fi
+
+if [[ $SANDBOX_SKIP_AUTHENTICATION == '1' ]]; then
```

Comment:
Instead of checking if it's an empty string like the other conditions  in this script, this condition checks against "1" because the `skip_authentication` parameter of the `test-branch` command in `.circleci/config.yml` sets the value to either "1" or "0". Thus, its value wouldn't be empty in the CI job.